src/Catel.MVVM/Catel.MVVM.Shared/Windows/Services/Interfaces/IWrapControlService.cs

```diff
@@ -0,0 +1,184 @@
+// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="IWrapControlService.cs" company="Catel development team">
+//   Copyright (c) 2008 - 2017 Catel development team. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Catel.Windows.Services
```

Comment:
Please just use Catel.Services instead of Catel.Windows.Services (especially if we are going to support other platforms).

I'm not 100 % sure how we are going to implement this one for other platforms, but I want to be prepared for the future without too many breaking changes.